packages/runner/src/run.ts

```diff
@@ -512,10 +513,12 @@ export async function startTests(paths: string[], runner: VitestRunner): Promise
   return files
 }
 
-async function publicCollect(paths: string[], runner: VitestRunner): Promise<File[]> {
+async function publicCollect(specs: FileSpec[], runner: VitestRunner): Promise<File[]> {
```

Comment:
This is still a public API, we need to allow `string[]` even if Vitest always passed down `FileSpec`